app/views/job_runs/_discovery_report_summary.html.erb

```diff
@@ -26,9 +26,11 @@
   </tbody>
 </table>
 
+<%= render partial: 'structural_updates_summary', locals: { druids: @discovery_report['rows'] } if @structural_has_changed %>
```

Comment:
@peetucket it looks like `@structural_has_changed` is not set to True in the turbo frame when it should be? 

For example see https://sul-preassembly-qa.stanford.edu/job_runs/877 which doesn't display the "Updates from csv" table. I tried removing the test and then it appeared...